cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -134,6 +134,15 @@ for k,v in yaml.load(sys.stdin).iteritems():
 ''' < "${kube_env_yaml}")
 fi
 
+# Wait for the default service account
+token_found=""
+while [ -z "${token_found}" ]; do
```

Comment:
Creeping elegance. This whole script is due for a rewrite in Go post-1.0, and is running async of the main startup block, so there's no opportunity to interact with anything. You could destroy the `/etc/motd` (which is what I changed the startup script to do to indicate things were good or not), but otherwise you don't have a lot of places to reach out to - we don't have an alerting system in place that you can hook into at this point.